server/@types/express/index.d.ts

```diff
@@ -77,14 +77,14 @@ export declare global {
 
     interface Response {
       redirectWithSuccess?(path: string, message: string): void
+      redirectWithErrors?(path: string, message: Record<string, string>[]): void
     }
 
     interface Request {
       verified?: boolean
       id: string
       logout(done: (err: unknown) => void): void
       flash(type: string, message: Array<Record<string, string>>): number
-      flash(message: 'errors'): Array<Record<string, string>>
```

Comment:
I'm pretty sure that this line came from the original template, so I'm wondering whether it would be as well to keep it, just in case there are future updates from the template in this area?